postgres-driver/loadbalancer.go

```diff
@@ -0,0 +1,59 @@
+package postgresdriver
```

Comment:
I think we also need a users package: 2 end-points that are specifically needed are: GetUsers and GetUserApps(userID) []Application